packages/bpk-component-map/stories.js

```diff
@@ -149,7 +149,7 @@ storiesOf('bpk-component-map', module)
   .add('With onTilesLoaded', () => (
     <StoryMap
       center={{ latitude: 55.944357, longitude: -3.1967116 }}
-      onTilesLoaded={() => console.log('Tiles loaded')}
+      onTilesLoaded={() => console.log('Tiles loaded')} // eslint-disable-line no-console
```

Comment:
Danger was complaining about this, so better to silent the warning in this case to avoid noise in all pr's 😅 

![image](https://user-images.githubusercontent.com/7152781/69343802-90976c00-0c6e-11ea-862f-bbfc1fd623ac.png)